src/components/aboutus/AboutUs.jsx

```diff
@@ -128,81 +224,32 @@ const AboutUs = () => {
-                                <div
-                                    key={idx}
-                                    className=' rounded-lg transition duration-300 ease-in-out transform hover:scale-105 sm:max-w-bg-white shadow-md p-2 bg-gray-200'
-                                >
-                                    <div className='w-24 h-24 mx-auto mb-3 rotate-vertical'>
-                                        <Image
-                                            src={item.avatar}
-                                            className='w-full h-full rounded-full'
-                                            alt=''
-                                            width={64}
-                                            height={64}
-                                        />
-                                    </div>
-
-                                    <h4 className='text-gray-700 font-semibold text-lg mb-2'>
-                                        {item.name}
-                                    </h4>
-                                    <p className='text-indigo-600 mb-2'>
-                                        {item.title}
-                                    </p>
-                                    <p className='text-gray-600 mb-4'>
-                                        {item.desc}
-                                    </p>
-                                    <div className='flex justify-center gap-4 mb-1'>
-                                        <div className='flex gap-2'>
-                                            <a
-                                                href={item.linkedin}
-                                                target='_blank'
-                                                rel='noopener noreferrer'
-                                            >
-                                                <FaLinkedin
-                                                    size={20}
-                                                    className='text-blue-500'
-                                                />
-                                            </a>
-                                            <a
-                                                href={item.GitHub}
-                                                target='_blank'
-                                                rel='noopener noreferrer'
-                                            >
-                                                <FaGithub
-                                                    size={20}
-                                                    className='text-gray-700'
-                                                />
-                                            </a>
-                                        </div>
-                                    </div>
-                                </div>
-                            ))}
-                        </div>
+                        <TeamComponent team={team} />
```

Comment:
not sure why it's not working !

Reply:
i will make sure to fix it, thank you for ur remarks